### PR TITLE
feat: add supabase completion spec

### PR DESCRIPTION
**Pull request by zeluizr**
What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)
Adds [supabase CLI](https://supabase.io/docs/reference/cli/getting-started) completion spec.

### CODE REVIEW

File: src/supabase.ts [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+const completionSpec: Fig.Spec = {
+  name: "supabase",
+  description:
+    "The CLI is still under heavy development, but it will contain all the functionality for working with Supabase projects and the Supabase platform",
+  subcommands: [
+    {
+      name: "init",
+      description:
+        "This command will create a .supabase folder which holds all the configuration for developing your project locally. You don't need to check this into version control",
+      options: [...commonOptions],
```
Comment (fedeci) [inline src/supabase.ts:18]: nit: a new array isn't necessary here. Same for the other occurrences in the file
```suggestion
      options: commonOptions,
```

**Review by fedeci — DISMISSED** [review]
Thanks!

**Comment by brendanfalk** [comment]
Let's hold off merging this for a few more days. Waiting to hear back from Supabase team what the most up to date docs are for the CLI
These: https://github.com/supabase/cli
or these: https://supabase.io/docs/guides/local-development#stop

**Comment by fedeci** [comment]
I've seen supabase is using our cobra integration, I think we can autogenerate a new updated spec based on the current `main` branch. Since this PR is stale since a while @ngajhede, would you like to create a new one with the changes?

**Comment by lyqht** [comment]
hey @fedeci, I'm thinking to creating a new updated spec for supabase CLI. may I ask more about the cobra integration and what you mean by autogenerating a spec based on this integration? Thank you!

**Comment by fedeci** [comment]
Ji @lyqht, this is what I meant: https://fig.io/docs/guides/integrating/integrations/cobra#fig-autocomplete--cobra

**Comment by fedeci** [comment]
Closing since a new PR is available

**Pull request closed without merging**